src/components/pages/HomePage.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+function HomePage() {
+  return (
+    <div>
+      <h1>Welcome to the Bookstore</h1>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore non
+        mollitia officia itaque iusto aspernatur doloremque magni voluptate
+        distinctio excepturi. Libero assumenda sed a, ex error tenetur accusamus
+        repudiandae, saepe enim optio reiciendis est id reprehenderit mollitia,
+        ipsum sunt corporis?
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore non
+        mollitia officia itaque iusto aspernatur doloremque magni voluptate
+        distinctio excepturi. Libero assumenda sed a, ex error tenetur accusamus
+        repudiandae, saepe enim optio reiciendis est id reprehenderit mollitia,
+        ipsum sunt corporis?
+      </p>
+      <p>
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore non
+        mollitia officia itaque iusto aspernatur doloremque magni voluptate
+        distinctio excepturi. Libero assumenda sed a, ex error tenetur accusamus
+        repudiandae, saepe enim optio reiciendis est id reprehenderit mollitia,
+        ipsum sunt corporis?
+        Lorem ipsum dolor sit amet consectetur adipisicing elit. Tempore non
+        mollitia officia itaque iusto aspernatur doloremque magni voluptate
+        distinctio excepturi. Libero assumenda sed a, ex error tenetur accusamus
+        repudiandae, saepe enim optio reiciendis est id reprehenderit mollitia,
+        ipsum sunt corporis?
+      </p>
+    </div>
+  );
+}
```

Comment:
- [ ] As per requirement this `component` is not needed navigation bar. should have `<Books>` and `<Categories>` only

Reply:
- [ ] Make sure this `Books` component contains the following components:
- `<BookList>`
- `<Book>`
- `<NewBookForm>`
That means when you open the application `Books` component should be the default page to be displayed